app/mailers/communicart_mailer.rb

```diff
@@ -7,9 +7,11 @@ class CommunicartMailer < ActionMailer::Base
   add_template_helper ClientHelper
 
 
-  def proposal_notification_email(to_email, approval, show_approval_actions=true)
+  def proposal_notification_email(to_email, approval,
+                                  show_approval_actions=true, alert_partial=nil)
```

Comment:
Come to think of it, the complex method signature might be a hint that we should split the pure notification email up from the one that includes the approval actions, even if they share most of the same logic/partials. Thoughts?